spec/rspec/mocks/and_call_original_spec.rb

```diff
@@ -185,6 +185,12 @@ class << self
       expect(klazz.alternate_new).to be_an_instance_of(klazz)
     end
 
+    it "works for methods that accept keyword arguments" do
+      def instance.foo(bar:); bar; end
+      expect(instance).to receive(:foo).and_call_original
+      expect(instance.foo(bar: "baz")).to eq("baz")
+    end
```

Comment:
This needs to be wrapped in an eval for our older ruby test suite, and skipped when theres no keyword argument support. (RSpec 3 supports 1.8.7 up).

Reply:
An example https://github.com/rspec/rspec-expectations/blob/955151da74fd6356e614e62addb3da26ea1ccaf6/spec/rspec/matchers/built_in/has_spec.rb#L20

Reply:
Ah, makes sense. Did that here: https://github.com/rspec/rspec-mocks/pull/1385/commits/1433a4299d726e236ab3d661fc54c73518b96e80